Remotion/Web/Core/ExecutionEngine/WxeUrlSettings.cs

```diff
@@ -58,7 +58,7 @@ public static WxeUrlSettings Create (string? urlMappingFile = null, int? maximum
 
     /// <summary> Gets the path to the file holding the URL mapping configuration. </summary>
     /// <value> A string. Defaults to 'UrlMapping.xml'. </value>
-    public string? UrlMappingFile { get; }
+    public string UrlMappingFile { get; }
```

Comment:
This should remain nullable. null means not here. Empty is not supported. Also, a follow-up task will remove this property completely so no need to change this here.